backend/src/Edna/Config/Environment.hs

```diff
@@ -110,3 +115,6 @@ dbInitialisationInitScriptEnv = lookupEnv "EDNA_DB_INITIALISATION_INIT_SCRIPT"
 
 loggingEnv :: IO (Maybe LoggingConfig)
 loggingEnv = parseEnv parseLoggingConfig "EDNA_LOGGING" WrongLogging
+
+mdeHostEnv :: IO (Maybe MdeHost)
+mdeHostEnv = lookupEnv "EDNA_MDE_HOST" <&> (T.pack <$>)
```

Comment:
In `Universum` there is `<<$>>` which is `fmap . fmap`, so I guess it can be written as `T.pack <<$>> lookupEnv "EDNA_MDE_HOST"`.